views/home.handlebars

```diff
@@ -1,4 +1,4 @@
-<h1>Organise all the things</h1>
+<h1>{{siteName}}</h1>
```

Comment:
This is one that I specifically changed the semantic meaning of to "site name", because it lets the "gath.io" branding coexist alongside whatever the name of your instance is.